source/content/guides/autopilot/04-troubleshoot.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Pantheon Autopilot
+subtitle: Troubleshoot Autopilot Error Messages
+description: "Diagnose and remedy some common Autopilot errors."
+categories: [automate]
+tags: [autopilot, troubleshoot, webops]
+type: guide
+layout: guide
+showtoc: true
+anchorid: troubleshoot-autopilot
+permalink: docs/guides/autopilot/troubleshoot-autopilot/
+editpath: autopilot/04-troubleshoot.md
+reviewed: "2021-07-30"
+---
+
+This page is a list of common issues and solutions that you may encounter when using Autopilot.
+
+## Error: Element Exclusions
+
+> We didn’t detect any plugins, modules or themes that were eligible for exclusion.
+
+### Issue
+
+Since the Autopilot plan is only set after Autopilot is initialized, the list of elements is unavailable when Autopilot is started for the first time.
```

Comment:
I want to reword the `Autopilot plan` part, but I'm not sure what to. Maybe about when the Autopilot feature is initialized? "Plan" sounds more internal.